src/JackTrip.cpp

```diff
@@ -1316,7 +1316,8 @@ int JackTrip::clientPingToServerStart()
         mAwaitingTcp = true;
         mElapsedTime = 0;
         mEndTime     = 30000;  // Timeout after 30 seconds.
-        mRetryTimer.setInterval(randomizer.bounded(0, 2000 * pow(2, mRetries)));
+        mRetryTimer.setInterval(randomizer.bounded(
+            static_cast<int>(0), static_cast<int>(2000 * pow(2, mRetries))));
```

Comment:
```suggestion
        mRetryTimer.setInterval(randomizer.bounded(0, 2000 * mRetries * mRetries));
```

Reply:
This should give us the desired behavior without the overhead of pow() and the potential casting and (some) bound errors.

Reply:
This crashes when using RtAudio/ALSA.

Reply:
I think the crash has nothing to do with this change.